schemas/layer-4.cue

```diff
@@ -1,35 +1,45 @@
 // Top level schema //
-
-"evaluation-plans": [...#EvaluationPlan]
+#FrameworkEvaluation: {
+    // name of the framework being evaluated
+    name: string
+    // ID of the framework being evaluated
+    frameworkID: string
+    // one or more evaluations of the framework controls
+    evaluations: [#ControlEvaluation, ...]
+}
 
 // Types
 
+#URL: =~"^https?://[^\\s]+$"
+
 #ControlEvaluation: {
+    // name of the control being evaluated
     name: string
-    "control-id": string
-    result: #Result
-    message: string
-    "documentation-url"?: =~"^https?://[^\\s]+$"
-    "corrupted-state"?: bool
-    "assessment-results"?: [...#AssessmentResult]
+    // ID of the control being evaluated
+    controlID: string
+    // one or more assessments of the control requirements
+    assessments: [#Assessment, ...]
 }
 
-#AssessmentResult: {
-    result: #Result
+#Assessment: {
+    // ID of the requirement being evaluated
+    requirementID: string
+    // the methods used to assess the requirement
+    methods: [#AssessmentMethod, ...]
+}
+
+#AssessmentMethod: {
+    // the method used to assess the requirement
     name: string
+    // detailed explanation of the method
     description: string
-    message: string
-    "function-address" string
```

Comment:
In the absence of documentation or existing usage, `function-address` should go away.